test/uk/gov/hmrc/agentservicesaccount/controllers/AgentServicesControllerSpec.scala

```diff
@@ -111,6 +111,16 @@ class AgentServicesControllerSpec extends BaseISpec {
       content should include(messagesApi("agent.services.account.heading"))
       content should include(messagesApi("app.name"))
       content should include(messagesApi("agent.accountNumber","TARN 000 0001"))
+//      *****19TH APRIL RELEASE *******
```

Comment:
Wondering if we set the feature flag in the tests, so that we don't have to wait to comment this back in?  Or just read the feature flag and include the test cases if it's set? Something along those lines, feels like the tests shouldn't need to wait for the app configuration to be set.

Reply:
yea you right, I am going to create another test suite with the feature flag on...